python/BioSimSpace/Sandpit/Exscientia/Protocol/_config.py

```diff
@@ -447,7 +450,10 @@ def generateGromacsConfig(self, extra_options=None, extra_lines=None):
             if self.protocol.getPressure() is not None:
                 # Don't use barostat for vacuum simulations.
                 if self._has_box and self._has_water:
-                    protocol_dict["pcoupl"] = "c-rescale"  # Barostat type.
+                    if _gmx_version >= 2021:
```

Comment:
Is this always safe?
Conda could give Gromacs version that is `2022.4-conda_forge`

Reply:
This looked like it should be fine based on:
 https://github.com/OpenBioSim/biosimspace/blob/f2acfc00cb5eb9675822d6dd57f2f4c463969013/python/BioSimSpace/Sandpit/Exscientia/__init__.py#L214

Reply:
Oh yeah, I added that bit.